src/transformers/modeling_utils.py

```diff
@@ -1528,13 +1872,15 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         elif from_pt:
 
             if low_cpu_mem_usage:
-                cls._load_state_dict_into_model_low_mem(model, loaded_state_dict_keys, resolved_archive_file)
+                cls._load_pretrained_model_low_mem(model, loaded_state_dict_keys, resolved_archive_file)
             else:
-                model, missing_keys, unexpected_keys, mismatched_keys, error_msgs = cls._load_state_dict_into_model(
+                model, missing_keys, unexpected_keys, mismatched_keys, error_msgs = cls._load_pretrained_model(
```

Comment:
Renamed to avoid confusion with the function `_load_state_dict_into_model` above. Also it does way more than loading the state dict.